modules/administration/pages/maintenance-windows.adoc

```diff
@@ -0,0 +1,142 @@
+====
+
+
+When you have created your maintenance windows, you can schedule restricted actions, such as package upgrades, to be performed during the maintenance window.
+
+
+
+.Procedure: Scheduling a Package Upgrade
+. In the {productname} {webui}, navigate to menu:Systems[System List], select the client you want to upgrade, and go to the menu:Software[Packages > Upgrade] tab.
+. Select the package to upgrade from the list, and click btn:[Upgrade Packages].
+. In the [guimenu]``Maintenance Window`` field, select which maintenance window the client should use to perform the upgrade.
+. Click btn:[Confirm] to schedule the package upgrade.
+
+
+
+== Maintenance Schedule Types
+
+When you create a calendar, it contains a number of events, which can be either one-time events, or recurring events.
+Each event contains a ``summary`` field.
+If you want to create multiple maintenance schedules for one calendar, you can specify events for each using the ``summary`` field.
+
+For example, you might like to create a schedule for production servers, and a different schedule for testing servers.
+In this case, you would specify ``SUMMARY: Production Servers`` on events for the production servers, and ``SUMMARY: Testing Servers`` on events for the testing servers.
+
+image::maint_windows_multi.png[scaledwidth=80%]
+
+There are two types of schedule: single, or multi.
+If your calendar contains events that apply to more than one schedule, then you must select ``multi``, and ensure you name the schedule according to the ``summary`` field you used in the calendar file.
+
+
+
+.Procedure: Creating a Multi Schedule
+. In the {productname} {webui}, navigate to menu:Schedule[Maintenance Windows > Schedules], and click btn:[Create].
+. In the [guimenu]``Schedule Name`` section, type the name for your schedule.
+Ensure it matches the ``summary`` field of the calendar.
+. Check the [guimenu]``Multi`` option.
+. Select the calendar to assign to this schedule.
+. Click btn:[Create Schedule] to save your schedule.
+. To create the next schedule, click btn:[Create].
+. In the [guimenu]``Schedule Name`` section, type the name for your second schedule.
+Ensure it matches the ``summary`` field of the second calendar.
+. Check the [guimenu]``Multi`` option.
+. Click btn:[Create Schedule] to save your schedule.
+. Repeat for each schedule you need to create.
+
+
+
+== Restricted and Unrestricted Actions
+
+This sections contains a complete list of restricted and unrestricted actions.
+
+Restricted actions significantly modify the client, and could potentially cause the client to stop running.
+Restricted actions can only be run during a maintenance window.
+The restricted actions are:
+
+* Package operations (for example, installing, updating, or removing packages)
+* Patch updates
+* Rebooting a client
+* Rolling back transactions
+* Configuration management changing tasks
+* Applying a highstate (for Salt clients)
+* Autoinstallation and reinstallation
+* Remote commands
```

Comment:
Maybe we should add a special note about remote command and salt.
We restrict only `actions` which can be scheduled. 

But for salt clients we have an additional page where you can run remote commands for it.
Salt => Remote Command  - here you can run a command directly. This works also for clients which are not in maintenance mode.